### PR TITLE
Js dom

### PR DESCRIPTION
**Pull request by Shrai-dev**
### Js dom

[Demo](https://shrai-dev.github.io/kottans-interactive-side-menu/)
[Code base](https://github.com/Shrai-dev/kottans-interactive-side-menu)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Shrai-dev/js_dom/main.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    title: "Casa Batlló",
+    src: "./assets/place5.jpg",
+    text:
+      "Yet another amazing Gaudí creation, the UNESCO-listed Casa Batlló is one of the most characteristic Modernist buildings in Barcelona. The fantastical mansion was designed as a private residence for the textile manufacturer Josep Batlló i Casanovas. With its freely swinging shapes and ornamental facade, this dreamlike building looks like a castle from a surreal fairy tale.",
+  },
+];
+const header = document.querySelector(".header");
+
+const headerTitle = document.createElement("h1");
+headerTitle.className = "header__title";
```
Comment (A-Ostrovnyy) [inline submissions/Shrai-dev/js_dom/main.js:46]: Use [classList](https://developer.mozilla.org/ru/docs/Web/API/Element/classList) API for manipulating with classes.

File: submissions/Shrai-dev/js_dom/main.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    const container = document.querySelector(".container__content");
+    const placeArray = places
+      .filter((place) => place.navTitle === placeNumber)
+      .map((place) => createContent(place));
+    placeArray.forEach((place) => container.appendChild(place));
+  });
+
+document.getElementById("trigger").onclick = function () {
+  open();
+};
```
Comment (A-Ostrovnyy) [inline submissions/Shrai-dev/js_dom/main.js:101]: Avoid adding eventListeners by ´onEvent´ use ´addEventListener´ instead.

File: submissions/Shrai-dev/js_dom/main.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  itemTitle.className = "content-title";
+  itemTitle.innerText = place.title;
+
+  const itemParagraph = document.createElement("p");
+  itemParagraph.className = "content-text";
+  itemParagraph.innerText = place.text;
+
+  const itemImage = document.createElement("img");
+  itemImage.className = "content-image";
+  itemImage.innerText = place.src;
```
Comment (A-Ostrovnyy) [inline submissions/Shrai-dev/js_dom/main.js:77]: InnerText for img tag?

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
See my comments below.

File: submissions/Shrai-dev/js_dom/main.js (hunk 1 above)
Comment (Shrai-dev) [reply submissions/Shrai-dev/js_dom/main.js]: Thanks for your comments! Everything is fixed.

Comment (A-Ostrovnyy) [reply submissions/Shrai-dev/js_dom/main.js]: Not fixed yet. [Read more](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) how to add class to the element using classList.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
One small issue)

File: submissions/Shrai-dev/js_dom/main.js (hunk 1 above)
Comment (Shrai-dev) [reply submissions/Shrai-dev/js_dom/main.js]: Done)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**